gazebo_plugins/CMakeLists.txt

```diff
@@ -316,8 +316,13 @@ target_link_libraries(gazebo_ros_template ${catkin_LIBRARIES} ${Boost_LIBRARIES}
 
 install(TARGETS
   hokuyo_node
-  vision_reconfigure
   camera_synchronizer
+  pub_joint_trajectory_test
```

Comment:
How about not installing this test anymore?

Reply:
Done.